### PR TITLE
Response helper

### PR DESCRIPTION
**Pull request by asoluwaseun**
#### What does this PR do?
This PR ensures API responses to clients have a consistent payload structure.

#### Description of task to be completed?
None

#### How should this manually be tested?
The function helper takes four parameters:

1. res - an Express response object.
2. status - HTTP status code.
3. error - a boolean flag for error (optional).
4. data - the response payload (optional).
5. message - response message (optional)

**Sample Use Case** 
```javascript
const { sendResponse } = require('../shared/responseHelper')

sendResponse(response, 200, false, user, "User details fetched.");
```

#### What are the relevant user stories?
None
#### Screenshots (if appropriate)
None

### CODE REVIEW

**Review by meetKazuki — CHANGES_REQUESTED** [review]
TBH, I don't think we need an abstraction for something like this at this stage.

File: src/shared/responseHelper.js [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+     * @param {Object} data Response data object
+     * @param {String} message Response message to client
+     *
+     * @returns {Object} Response from the server
+     */
+
+    static sendResponse(
+        res,
+        status,
+        error = false,
```
Comment (tag6071) [inline src/shared/responseHelper.js:18]: I think this is not needed, it is kind of redundant, instead what would be ok is having an error object

File: src/shared/responseHelper.js [hunk 2]
```diff
@@ -0,0 +1,69 @@
+class ResponseHelper{
```
Comment (tag6071) [inline src/shared/responseHelper.js:1]: The whole implementation could be refactored to this 👇 

```.js
class ResponseHelper {
  /**
   * @function
   * @description A simple API response helper
   *
   * @param {Object} res Express response object
   * @param {Number} status HTTP status code
   * @param {Boolean} error Flag for error
   * @param {Object} data Response data object
   * @param {String} message Response message to client
   *
   * @returns {Object} Response from the server
   */


  static sendResponse(res, status, data = {}, message = '') {
    const statusObj = {
      200: 'Successful',
      201: 'Created Successfully',
      204: ':) hmmm',
      400: 'Invalid Request Format',
      401: 'Unauthorized Access',
      403: 'Forbidden',
      404: 'Resource Not Found',
      405: 'Method Not Allowed',
      500: 'Internal Server Error'
    };

    let response = { data: data || null, error };

    if (message) {
      statusObj[status] = message;
    }

    return res.status(status).json(response);
  }
}
```

**Review by tag6071 — COMMENTED** [review]
Good Job @asoluwaseun please see my comments.

I agree with @meetKazuki as I think this is not needed now at this point of the project, we should wait till there is an actual implementation where `request` and `response` would be done etc, then we can decide what or if to use a specification, there is OpenAPI, JSON API and maybe more, I think we would need to agree on this. You could setup a call later for us to discuss this.

**Pull request merged**